### PR TITLE
Update alpine version in image vector

### PR DESCRIPTION
**Pull request by vpnachev**
**How to categorize this PR?**

**What this PR does / why we need it**:
Update alpine version in image vector.

**Which issue(s) this PR fixes**:
Fixes #

**Special notes for your reviewer**:

**Release note**:

```other operator

```

### CODE REVIEW

File: charts/images.yaml [hunk 1]
```diff
@@ -190,7 +190,7 @@ images:
 # Miscellaenous
 - name: alpine
   repository: alpine
-  tag: "3.10.3"
+  tag: "3.13.5"
```
Comment (ialidzhikov) [inline charts/images.yaml:193]: I guess it is better to update this with `alpine-iptables` together. @DockToFuture , can you please push `3.13.5` tag for the alpine-iptables image?

Comment (vpnachev) [reply charts/images.yaml:193]: It is a bit unfortunate that the repository https://github.com/DockToFuture/alpine-iptables no longer exist.
And we had discussion about this image long time ago here https://github.com/gardener/gardener/pull/1644.

Comment (rfranzke) [reply charts/images.yaml:193]: FYI: There is #4574 now.

**Review by rfranzke — APPROVED** [review]
/lgtm

**Pull request closed without merging**